app/src/app/[lng]/[inventory]/InventoryResultTab/EmissionsForecast/GrowthRatesExplanationModalTable.tsx

```diff
@@ -16,22 +17,36 @@ export const GrowthRatesExplanationModalTable = ({
     return sector.name === "ippu" ? sector.name + "-short" : sector.name;
   }
 
+  // for emission growth rates, sector V is split between its subsectors
+  const emissionsSectors = [
+    ...SECTORS.slice(0, 4),
+    ...Object.values(SECTORS[4]!.subSectors!),
+  ];
+
   return (
-    <Table.Root unstyled>
+    <Table.Root>
       <Table.Header>
         <Table.Row>
-          <Table.ColumnHeader>{t("sector")}</Table.ColumnHeader>
+          <Table.ColumnHeader>
+            <ButtonSmall textTransform="uppercase">{t("sector")}</ButtonSmall>
+          </Table.ColumnHeader>
           {Object.keys(growthRates)
             .slice(0, 4)
             .map((year) => (
-              <Table.ColumnHeader key={year}>{year}</Table.ColumnHeader>
+              <Table.ColumnHeader key={year}>
+                <ButtonSmall textTransform="uppercase">{year}</ButtonSmall>
+              </Table.ColumnHeader>
             ))}
-          <Table.ColumnHeader>{"2030"}</Table.ColumnHeader>
-          <Table.ColumnHeader>{"2050"}</Table.ColumnHeader>
+          <Table.ColumnHeader>
+            <ButtonSmall textTransform="uppercase">{"2030"}</ButtonSmall>
+          </Table.ColumnHeader>
+          <Table.ColumnHeader>
+            <ButtonSmall textTransform="uppercase">{"2050"}</ButtonSmall>
```

Comment:
```suggestion
            <ButtonSmall>2030</ButtonSmall>
          </Table.ColumnHeader>
          <Table.ColumnHeader>
            <ButtonSmall>2050</ButtonSmall>
```
This would be sufficient